src/System.Runtime.Extensions/tests/System/AppDomainTests.netcoreapp.cs

```diff
@@ -197,5 +201,49 @@ public static IEnumerable<object[]> TestingCreateInstanceObjectHandleFullSignatu
             yield return new object[] { "AssemblyResolveTestApp", "AssemblyResolveTestApp.PrivateClassSample", false, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[1] { 1 }, CultureInfo.InvariantCulture, null, "AssemblyResolveTestApp.PrivateClassSample" };
             yield return new object[] { "assemblyresolvetestapp", "assemblyresolvetestapp.privateclasssample", true, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, Type.DefaultBinder, new object[1] { 1 }, CultureInfo.InvariantCulture, null, "AssemblyResolveTestApp.PrivateClassSample" };
         }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Does not support AssemblyLoadContext")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Does not support AssemblyLoadContext")]
```

Comment:
I think this attribute should not be needed as this ".netcoreapp.cs" file is only included when building tests for netcoreapp. We could not build it against NETFX anyway.